changelog/pending/20230128--backend-filestate--the-filestate-backend-now-supports-project-scoped-stacks.yaml

```diff
@@ -0,0 +1,7 @@
+changes:
+- type: feat
+  scope: backend/filestate
+  description: |
+    Add support for project-scoped stacks.
```

Comment:
We should add a comment here that write access to the root of the bucket is now needed to write "meta.yaml"